src/MyService/Program.cs

```diff
@@ -1,22 +1,43 @@
-﻿using NServiceBus;
+﻿using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using NServiceBus;
 using System;
-using System.Threading.Tasks;
+using Serilog;
 
 namespace MyService
 {
-    class Program
+    public class Program
     {
-        static async Task Main(string[] args)
+        public static void Main(string[] args)
         {
-            Console.Title = typeof(Program).Namespace;
+            CreateHostBuilder(args).Build().Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args, Action<EndpointConfiguration> configPreview = null)
+        {
+            var builder = Host.CreateDefaultBuilder(args);
+            builder.UseConsoleLifetime();
+            
+            builder.UseSerilog((context, services, loggerConfiguration) => loggerConfiguration
```

Comment:
I am not using the [Inline initialization](https://github.com/serilog/serilog-extensions-hosting#inline-initialization) but the [explicit one](https://github.com/serilog/serilog-extensions-hosting#instructions). But this shouldn't make a difference.